CMakeLists.txt

```
@@ -610,6 +612,19 @@ if(CURL_USE_GNUTLS)
   endif()
 endif()
 
+if(CURL_USE_RUSTLS)
+  find_package(rustls REQUIRED)
+  set(_ssl_enabled ON)
+  set(USE_RUSTLS ON)
+  list(APPEND CURL_LIBS ${RUSTLS_LIBRARY})
```

```suggestion
  list(APPEND CURL_LIBS ${RUSTLS_LIBRARIES})
```
Just formalities, to stay with the conventions we're aiming at: #14529